readthedocs/rtd_tests/tests/test_celery.py

```diff
@@ -39,6 +42,11 @@
 )
 
 
+# NOTE: most of these tests need to be re-written making usage of the new
+# Celery handlers. These are exactly the tests we are interested in making them
+# work properly (e.g. send notifications after build, handle unexpected
+# exceptions, etc)
+@pytest.mark.skip
```

Comment:
Also just a marker to make sure we don't lose this point. This seems important for testing, but I know this is still WIP.